src/views/PositionPdfPlot/PositionPdfPlotWidget.tsx

```diff
@@ -55,13 +62,7 @@ const usePositionPdfDataModel = (fetchSegment: (q: FetchSegmentQuery) => Promise
         }
         return ret
     }, [fetchSegmentCache, numPositions, segmentSize])
-    return {
-        get,
-        numTimepoints,
-        numPositions,
-        segmentSize,
-        multiscaleFactor
-    }
+    return { get }
```

Comment:
The other parts of this were actually just being passed through from the function inputs, so I reference them directly to avoid any memoization tricks.